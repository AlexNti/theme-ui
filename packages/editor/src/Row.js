/** @jsx jsx */
import { jsx } from 'theme-ui'

// todo: replace with Grid?
export default ({ width = 64, ...props }) => (
  <div
    {...props}
    sx={{
      display: 'grid',
      gridGap: 2,
      gridTemplateColumns: `repeat(auto-fit, minmax(${width}px, 1fr))`,
    }}
  />
)
