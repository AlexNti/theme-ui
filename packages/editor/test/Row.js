import React from 'react'
import renderer from 'react-test-renderer'
import Row from '../src/Row'

test('renders', () => {
  const json = renderer.create(
    <Row />
  ).toJSON()
  expect(json).toMatchSnapshot()
})
