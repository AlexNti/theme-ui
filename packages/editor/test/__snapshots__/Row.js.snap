// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.emotion-0 {
  display: grid;
  grid-gap: 8px;
  grid-template-columns: repeat(auto-fit,minmax(64px,1fr));
}

<div
  className="emotion-0"
/>
`;
